lib/insights_cloud/async/insights_full_sync.rb

```diff
@@ -79,8 +95,19 @@ def to_model_hash(hit_hash)
           total_risk: hit_hash['total_risk'].to_i,
           likelihood: hit_hash['likelihood'].to_i,
           results_url: hit_hash['results_url'],
+          rule_id: to_rule_id(hit_hash['results_url']),
```

Comment:
can we also add `has_playbook` data here based on the `rule_id`?

Reply:
At this point I don't know which rules have playbooks, since I didn't fetch recommendations yet.